tests/assets/workflow_example/daylightfactor.yaml

```diff
@@ -64,104 +82,123 @@ templates:
+          location: project-folder
+          path: output/raw/
+          task_path: results-folder
+    command: honeybee radiance grid merge results-folder results .res --folder out
     outputs:
       artifacts:
         - name: result-file
-          path: "{{inputs.parameters.output-folder}}/{{inputs.parameters.base-name}}.res"
+          location: project-folder
+          path: output/postprocess/results.res
+          task_path: "out/results.res"
 
 flow:
   name: daylight-factor
   tasks:
     - name: generate-overcast-sky
       template: generate-overcast-sky
-      arguments:
-        parameters:
-          - name: project-folder
-            value: '{{ workflow.inputs.parameters.project-folder }}'
 
     - name: split-grid
       template: split-grid
       arguments:
         parameters:
           - name: sensor-count
             value: "{{ workflow.inputs.parameters.sensor-count }}"
-          - name: input-grid
-            value: "{{ workflow.inputs.parameters.grid-name }}"
-          - name: output-folder
-            value: "{{ workflow.inputs.parameters.project-folder }}"
+
     - name: create-octree
       template: create-octree
       dependencies:
         - generate-overcast-sky
       arguments:
-        - name: sky-file
-          value: "{{ tasks.generate-sky.outputs.artifacts.sky }}"
-        - name: scene-files
-          value: "{{ workflow.inputs.parameters.scene-files }}"
+        artifacts:
+          - name: sky-file
+            path: "{{ tasks.generate-sky.outputs.artifacts.sky.path }}"
+
 
     - name: trace-rays
-      template: trace-rays
+      template: trace-rays-daylight-factor
       dependencies:
         - create-octree
         - split-grid
       arguments:
         parameters:
-          - name: grid
-            value: '{{ item }}'
-          - name: octree
-            value: "{{ tasks.create-octree.outputs.artifacts.octree-file }}"
+          - name: ray-tracing-options
+            value: '-ab 2'
+      artifacts:
+        - name: octree-file
+          path: "{{tasks.generate-octree.outputs.artifacts.octree-file}}"
+        - name: grid-file
+          path: "asset/grid/{{item}}" # item represents the individual string output from looping over output grids below
       loop: "{{tasks.generate-sky.outputs.parameters.split-grid}}"  # loop over the output grids
 
     - name: merge-results
       template: merge-results
       dependencies:
         - trace-rays
-      arguments:
-        parameters:
-          - name: base-name
-            value: "{{ workflow.inputs.parameters.grid-name }}"
 
 # this is not supported in Argo or is it?
-outputs:
-  artifacts:
-    - name: daylight-factor-values
-      path: "{{ tasks.merge-results.outputs.artifacts.result-file }}"
+# outputs:
+#   artifacts:
+#     - name: daylight-factor-values
+#       path: "{{ tasks.merge-results.outputs.artifacts.result-file }}"
```

Comment:
Why are we commenting this out?

Reply:
That's my bad! I left it out because it was confusing me during the testing process. Forgot to uncomment.